azavea/echo-locator
# Fix overweighting crime

## Overview

Fixes a neighborhood weighting/sorting issue when crime is marked "very important."

Also makes some minor package updates.


## Demo

Before:

![echolocator_sorting_before](https://user-images.githubusercontent.com/960264/77187917-95005400-6aab-11ea-86ee-165da4ea4522.png)

After:

![echolocator_sorting_after](https://user-images.githubusercontent.com/960264/77187929-9af63500-6aab-11ea-8e5c-75975b44f82c.png)


## Testing Instructions

 * Add profile address 1050 Adams Street, Dorchester Center (or use profile `22222222`)
 * Sorting should behave differently than on staging
 
Fixes #321
